www/js/intro.js

```diff
@@ -45,7 +45,7 @@ angular.module('emission.intro', ['emission.splash.startprefs',
   };
 
   $scope.disagree = function() {
-    $state.go('root.main.heatmap');
+    $state.go('intro.refuse');
```

Comment:
you also need a new state configuration of this.
you may want to rename this to 'root.main.refuse' and use something similar to 
https://github.com/e-mission/e-mission-phone/blob/master/www/js/main.js#L34